workspace/scm.py

```diff
@@ -193,6 +193,15 @@ def merge_branch(branch, squash=False, strategy=None):
         cmd.append('--squash')
     if strategy:
         cmd.append('--strategy=' + strategy)
+
+    current = current_branch()  # uses current_branch() method  to get current branch
+    message = ""
+    if current:
+        message = "Merge branch %s into %s" % (branch, current)
+    if strategy:
+        message = message + "\nUsing strategy " + strategy
+    cmd.append('-m ' + message)
```

Comment:
use format() to create this string. i think you also need to quote the message otherwise when it sees a space in the commit message it will consider the next thing to be an argument.

Reply:
Looks like it works correctly without quote. If quotes are added, it will also add the quotes to the commit msg in git.

I would suggest appending using the following format instead of a newline so everything is in a single line: `message += f" (using strategy {strategy})"`